test/models/test_inspection.py

```diff
@@ -75,13 +87,13 @@ def test_wrong_product_matchrules(self):
     self.assertFalse(self.inspection.validate())
```

Comment:
I wonder why the test does not fail here. Shouldn't validate always return `None`?